numpy/linalg/tests/test_linalg.py

```diff
@@ -539,11 +534,6 @@ def test_types(self, dtype):
         x = np.array([[1, 0.5], [0.5, 1]], dtype=dtype)
         assert_equal(linalg.inv(x).dtype, dtype)
 
-    @pytest.mark.xfail(sys.platform == 'cygwin',
-                       reason="Consistently fails on CI.")
-    def test_sq_cases(self):
```

Comment:
These are marking tests broken by OpenBLAS 0.3.19, but since this PR forces the CI to use netlib reference BLAS, those tests no longer fail.  If you want to keep them marked for local tests run with OpenBLAS, I can make another PR to deal with that, which would basically be a copy of #20654.

Reply:
It looks like the `def test_...` also got deleted, that is what I missed?

Reply:
The body of the test just calls the implementation from the superclass; if there is no mark on the function definition, then there is no need to have a function that manually does what inheritance already does.

The mark was there because these tests fail on AVX-512 with OpenBLAS 0.3.19, since I wanted to mark the test as `xfail` only in the case where it actually failed, I had to define a function with the same name in the class where it failed, then apply the decorator to the new function.  Calling the superclass implementation seemed to be the simplest method of keeping everything straight, and verifying that this doesn't actually change the behavior of the class.